credential/config.go

```diff
@@ -0,0 +1,33 @@
+/*
+ * Nuts node
+ * Copyright (C) 2021 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package credential
+
+const (
+	moduleName = "Verifiable credential Store"
```

Comment:
I like the similarity with 'VDR' (altough that should be acronymed to "Verifiable DID Registry" imo).

However if this is the module name, should we rename just the engine/type to 'VCS' instead of calling it "credential"? Now every time you name a variable "credential" (which is often, I'd expect) it collides with the package name. And I'd expect a `Credential` type to be, well, a credential. Not something that reads and writes credentials (because what would it be returning, credentials?)

Reply:
good suggestion